cabal-install/Distribution/Client/SetupWrapper.hs

```diff
@@ -365,7 +365,7 @@ runSetup verbosity setup args0 = do
 -- verbosity applies to ALL commands.
 verbosityHack :: Version -> [String] -> [String]
 verbosityHack ver args0
-    | ver >= mkVersion [1,25] = args0
+    | ver >= mkVersion [2,1]  = args0
```

Comment:
I asked @ezyang, and he told me

> I know why SetupWrapper has the hack: it's because verbosity flags apply to commands that are not configure
> but filterConfigureFlags is configure only 
> So, I think everything will still work even if you don't filterConfigureFlags. But it's "good style", I suppose

I'll update this PR to have `filterConfigureFlags` follow the good style... :)